src/main/java/com/cloudogu/smeagol/repository/domain/Branch.java

```diff
@@ -0,0 +1,49 @@
+package com.cloudogu.smeagol.repository.domain;
+
+import com.google.common.base.MoreObjects;
+
+import java.util.Objects;
+
+/**
+ * Branch represents an named branch of a repository.
+ */
+public class Branch {
+
+    private RepositoryId repositoryId;
+    private Name name;
+
+    public Branch(RepositoryId repositoryId, Name name) {
+        this.repositoryId = repositoryId;
+        this.name = name;
+    }
+
+    public RepositoryId getRepositoryId() {
+        return repositoryId;
+    }
+
+    public Name getName() {
+        return name;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS00122)